doc/run_experiment.rst

```diff
@@ -507,7 +507,9 @@ fixed_parameters *(Optional)*
 
 List of dicts containing parameters you want to have fixed for each
 classifier in :ref:`learners` list. Any empty ones will be ignored
-(and the defaults will be used).
+(and the defaults will be used). If there is a conflict between parameters
+specified in :ref:`param_grids` and :ref:`fixed_parameters`, :ref:`param_grids`
+values will take precedence.
```

Comment:
Isn't this the opposite of what we finally agreed on (after @dan-blanchard's [comment](https://github.com/EducationalTestingService/skll/pull/297#issuecomment-185818068)?